py/vttest/environment.py

```diff
@@ -28,7 +28,14 @@
 
 def get_test_directory():
   """Returns the toplevel directory for the tests. Might create it."""
-  directory = tempfile.mkdtemp(prefix='vt')
+  dataroot = os.environ['VTDATAROOT']
+  if dataroot:
+    directory = os.path.join(dataroot, "test")
```

Comment:
tempfile.mkdtemp takes a prefix, see below, but also a base dir (dir=...) so you could use that?
I think all these lines 31 to 38 could be just:
  directory = tempfile.mkdtemp(prefix='vttest', dir=os.environ.get('VTDATAROOT', None))